zerver/tests/tests.py

```diff
@@ -404,6 +404,7 @@ def test_api(self):
         result = self.client_delete('/json/bots/hamlet@zulip.com')
         self.assert_json_error(result, 'No such bot')
 
+@skip_py3
 class BotTest(AuthedTestCase):
```

Comment:
I got failures on Travis CI for many methods from this class, but they run fine on my computer. I couldn't find out the reason for this.